code/modules/mob/living/basic/retaliate/ghost.dm

```diff
@@ -107,3 +107,19 @@
 
 /datum/ai_behavior/basic_melee_attack/ghost
 	action_cooldown = 2 SECONDS
+
+
+/// Weaker variant of ghosts that can smash structures. Meant to be summoned in swarms via the ectoplasmic anomaly and associated ghost portal.
+
+/mob/living/basic/ghost/swarm
+	name = "vengeful spirit"
+	desc = "Oh my god, they're EVERYWHERE."
```

Comment:
This desc doesn't make any sense seeing because there's an instance where if they break away from the horde and you examine it... feels silly. Write the desc pertinent to this mob as if only one existed imo.